charts/argo-cd/templates/argocd-applicationset/deployment.yaml

```diff
@@ -43,6 +43,9 @@ spec:
       securityContext:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- with .Values.applicationSet.priorityClassName }}
```

Comment:
priorityClassName is not consistent across all yamls

Reply:
Default has been fixed.